core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -71,6 +77,7 @@ final case class PartKeyLuceneIndexRecord(partKey: Array[Byte], startTime: Long,
 
 class PartKeyLuceneIndex(ref: DatasetRef,
                          schema: PartitionSchema,
+                         facetEnabled: Boolean,
```

Comment:
Nit pick, Personally I prefer adding new fields at the end with default values, ``false`` in this case.

Reply:
I did not use default param values since I really wanted callers to think about faceting for performance.